internal/rootcoord/root_coord.go

```diff
@@ -2007,3 +2007,96 @@ func (c *Core) GetMetrics(ctx context.Context, req *milvuspb.GetMetricsRequest)
 		Response: "",
 	}, nil
 }
+
+func (c *Core) CreateAlias(ctx context.Context, in *milvuspb.CreateAliasRequest) (*commonpb.Status, error) {
+	code := c.stateCode.Load().(internalpb.StateCode)
+	if code != internalpb.StateCode_Healthy {
+		return &commonpb.Status{
+			ErrorCode: commonpb.ErrorCode_UnexpectedError,
+			Reason:    fmt.Sprintf("state code = %s", internalpb.StateCode_name[int32(code)]),
+		}, nil
+	}
+	log.Debug("CreateAlias ", zap.String("alias", in.Alias), zap.String("name", in.CollectionName))
+	t := &CreateAliasReqTask{
+		baseReqTask: baseReqTask{
+			ctx:  ctx,
+			core: c,
+		},
+		Req: in,
+	}
+	err := executeTask(t)
+	if err != nil {
+		log.Debug("CreateAlias failed", zap.String("alias", in.Alias), zap.String("name", in.CollectionName), zap.Error(err))
+		return &commonpb.Status{
+			ErrorCode: commonpb.ErrorCode_UnexpectedError,
+			Reason:    "Create alias failed: " + err.Error(),
+		}, nil
+	}
+	log.Debug("CreateAlias Success", zap.String("alias", in.Alias), zap.String("name", in.CollectionName))
+	return &commonpb.Status{
+		ErrorCode: commonpb.ErrorCode_Success,
+		Reason:    "",
+	}, nil
+}
+
+func (c *Core) DropAlias(ctx context.Context, in *milvuspb.DropAliasRequest) (*commonpb.Status, error) {
+	code := c.stateCode.Load().(internalpb.StateCode)
```

Comment:
What happens if we drop the collection? are we also cleaned up all the alias?

Reply:
I think we have to discuss this behavior, and hopefully, implement the functionality in a separate PR. 

We can either:
1. drop all the aliases when the user drops the collection & this could be destructive in a multi-user/client scenario.
2  suspend the user from dropping the collection if the collection is aliased.  RootCoordinator will ask the user to drop the alias first, after user drops the aliases, the user can drop the collection. 

What is your thought on this?

Reply:
Add a parameter "force_drop" in drop_collection() maybe a solution. If the force_drop=true, the api will drop the collection's alias automatically, otherwise, it will return an error to ask user to drop alias first once the collection has alias.

Reply:
Ok. I've added functionality to drop the aliases when dropping the collection.